api/types/installers/installer.sh.tmpl

```diff
@@ -110,6 +114,7 @@ on_gcp() {
   sudo /usr/local/bin/teleport node configure \
     --proxy="{{ .PublicProxyAddr }}" \
     --join-method=${JOIN_METHOD} \
+    --join-params="${JOIN_PARAMS}" \
```

Comment:
Can we not introduce a basically free-form arbitrary "join params" field that's easy to abuse IMO, and instead add a specific field for the Azure client ID?